README.md

```diff
@@ -1,6 +1,79 @@
-oic-auth
-============
+# oic-auth
 
 A Jenkins plugin which lets you login to Jenkins using your own, self-hosted or public openid connect server.
 
-Jenkins plugin website: https://plugins.jenkins.io/oic-auth
+[![Plugin Version](https://img.shields.io/jenkins/plugin/v/oic-auth.svg)](https://plugins.jenkins.io/oic-auth)
+[![Build Status](https://ci.jenkins.io/buildStatus/icon?job=Plugins/oic-auth-plugin/master)](https://ci.jenkins.io/job/Plugins/oic-auth-plugin/master)
+
+![OpenID connect](/docs/images/openid-connect-logo.jpg)
+
+## Open Tickets (bugs and feature requests)
+
+[https://github.com/jenkinsci/oic-auth-plugin/issues](https://github.com/jenkinsci/oic-auth-plugin/issues?q=is%3Aopen+is%3Aissue)
+
+## Configuration
+
+Configuration of this plugin takes a bit of effort as it requires some
+knowledge of the openid connect standard as well as the non-standard
+configuration of the various identity providers out there. Should you
+configure this plugin against a identity provider then please share your
+experiences and found caveats through a blog post or by adding it to the
+documentation.
+
+Also note that the spec describes a well known configuration location
+which will also help discovering your settings
+(<https://openid.net/specs/openid-connect-discovery-1_0.html#ProviderConfig>)
+
+From 1.5 and onward the well known configuration location may be used to
+populate the configuration simplifying the configuration greatly. See
+also the following screenshot utilizing the google well known endpoint
+for a minimal configuration example: 
+
+![global-config](/docs/images/global-config.png)
+
+### Using g-suite / google
+
+Obtain the client id and secret from the developer console: https://console.cloud.google.com/apis/credentials by creating Oauth client id.
+
+Use those to fill the respective fields in the configuration of Jenkins.
+
+Choose automatic configuration:
+
+Well-known configuration: https://accounts.google.com/.well-known/openid-configuration
+
+see also: <https://developers.google.com/identity/protocols/OpenIDConnect>
```

Comment:
And you should also set **Full name field name** to `name` and **Email field name** to `email` (weird that these are not defaults, since https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims defines them).